app.js

```diff
@@ -4,6 +4,10 @@
 const express = require('express');
 
 const app = express();
+const bodyParser = require('body-parser');
+
+app.use(bodyParser.json()); // for parsing application/json
+app.use(bodyParser.urlencoded({extended: true})); // for parsing application/x-www-form-urlencoded
```

Comment:
When do we have `application/x-www-form-urlencoded` encoded bodies? It should be false here otherwise.